Exceptions/PESEL/src/main/java/WrongTypeOfDataException.java

```diff
@@ -0,0 +1,5 @@
+public class WrongTypeOfDataException extends Exception {
+    public WrongTypeOfDataException() {
+        System.out.println("Wrong type of data provided");
```

Comment:
Ten sysout jest zbędny. Nie wnosi nic. Jak już to lepiej przypisać do pola np. 'message' więcej informacji o błędzie. Aktualnie printujesz na konsole to samo co użytkownik już wie (bo z nazwy exceptiona domyślić się można, że 'Wrong type of data'). Jeżeli byłoby tu wiecej informacji np. co jest nie tak z ta daną to wtedy spoko, ale sama informacja 'niepoprawne dane' nic nie mowi :P